### PR TITLE
[IMP] theme_buzzy: adapt s_timeline

### PR DESCRIPTION
**Pull request by Brieuc-brd**
This commit adapts the s_timeline customizations since this snippet has been redesigned in PR[1].

task-3657763
Part of task-3619705

Requires:
- https://github.com/odoo/odoo/pull/172720

[1]: https://github.com/odoo/odoo/pull/172720

### CODE REVIEW

**Review by stefanorigano — APPROVED** [review]
All good, thanks 👍

File: theme_buzzy/views/new_page_template.xml [hunk 1]
```diff
@@ -87,16 +87,13 @@
 </template>
```
Comment (sobo-odoo) [inline theme_buzzy/views/new_page_template.xml:87]: Commit message:
- the commit message should start with a verb :wink: => adapt s_timeline
- nitpick: space between PR et [1] :slightly_smiling_face:

**Review by sobo-odoo — CHANGES_REQUESTED** [review]
Here is a small commit message review :wink:

**Review by sobo-odoo — APPROVED** [review]
@robodoo r+

**Pull request closed without merging**